CONTRIBUTING.md

```diff
@@ -7,14 +7,20 @@ repositories will go through.  All changes, regardless of whether they are from
 newcomers to the community or from the core team follow the exact
 same process and are given the same level of review.
 
-- [Contributor License Agreements](#contributor-license-agreements)
+- [Code of conduct](#code-of-conduct)
+- [Contributor license agreements](#contributor-license-agreements)
 - [Issues](#issues)
 - [Contributing a feature](#contributing-a-feature)
-- [Pull Requests](#pull-requests)
+- [Pull requests](#pull-requests)
 
-## Contributor License Agreements
+## Code of conduct
 
-We'd love to accept your patches! But before we can take them, you will have 
+All members of the Istio community must abide by the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
```

Comment:
minor thing, but can you put a CR before the link so things fit within 80 chars. Some of us don't have those fancy huge monitors  ;-)   same for rest of PR